### DISCUSSION TITLE
Rationale for CC license?

### DISCUSSION THREAD

**Discussion by joliss** [microsoft/pylance-release#5008] [General]
I'm asking this mostly out of curiosity, not to debate the choice of license!

It seems that when this project was started, `LICENSE` [was added](https://github.com/microsoft/pylance-release/commit/43f62d32dca50308bd3db4c828e2c990d1401a57) with CC BY 4.0. Creative Commons licenses are pretty unusual for an open source project, so I'm wondering if there's any particular reason for this?

(An MIT `LICENSE-CODE` was also [initially added](https://github.com/microsoft/pylance-release/commit/203fae31a708c30f5ab08b9b8f868bab558059c0), but it was [removed](https://github.com/microsoft/pylance-release/commit/231e8d5addb1082792ece11d241ed2ff6e337d4b) soon after.)

**Comment by deepdive101** [comment]
For those encumbered by the license, `pyright` is a more open option.
